PROCESS.md

```diff
@@ -0,0 +1,163 @@
+# JSON Schema Specification Development Process
+This document formally defines the process used to evolve the JSON Schema
+specification as of the first stable release in 2023. It applies to the core
+specification and the specifications for any dialects and vocabularies
+maintained by the JSON Schema Org. It does not necessarily apply to everything
+the JSON Schema Org maintains. For example, media type registration follows the
+IETF process. Certain components used within JSON Schema, such as Relative JSON
+Pointer, may also follow a separate process.
+
+## Canonical URLs
+There MUST be a stable canonical URL for referencing any specification that
+follows this process. If the spec is made available from any other URLs, they
+SHOULD redirect to the canonical URL. If the canonical URL is changed in the
+future, all previous canonical URLs MUST remain accessible as redirects to the
+current URL.
+
+## Mutable Spec
+There is only one version of the specification and it can be updated at any time
+as long as those changes don't violate the compatibility rules identified by the
+stability model. Bug fixes, clarifications, and other non-functional spec
+updates can be merged at any time even for stable features. A change-log blog
+post SHOULD be prepared quarterly to give visibility to any changes that were
+merged in the last quarter.
+
+## The Stability Model
+Every feature in the spec has a release status. It's either evolving, stable, or
+deprecated. Flags are used to show the status a feature is in. If a feature
+doesn't have a flag, it's considered stable. If it has a year flag (such as
+`2024`) that's the year the feature reached stability. The other flags indicate
+a feature that is not yet stable or is deprecated.
+
+### Champions
+Any feature that is evolving SHOULD have a champion. The champion is responsible
+for shepherding the feature through the process to into stable status. The
+champion doesn't need to do all the work, they're just responsible for making
+sure the work gets done. The champion role MAY be shared by more than one
+person. An evolving feature without a champion SHOULD be removed from the spec.
+
+### Proposals
+New features start out as proposals. Proposals are informal GitHub Issues or
+Discussions and can be submitted by anyone.
+
+All proposals MUST be backward compatible with the current stable specification.
+Features that could be problematic for forward compatibility are highly
+discouraged, but may be considered if the feature is optional and disabled by
+default.
+
+**Promotion Checklist:**
```

Comment:
I assume these checklists are for promotion _out of_ this stage to the next one?

Reply:
Correct. Any suggestions to make that more clear?

Reply:
Maybe explicitly say "Checklist for promotion to stage-X"